### PR TITLE
make iter_lines deal with decoding errors during iteration; Also...

### PR DESCRIPTION
**Pull request by koreno**
remove broken support for no-decoding

### CODE REVIEW

**Comment by koreno** [comment]
It fails on that same pypy cryptography issue, unrelated to the code changes

**Comment by henryiii** [comment]
Restarting the CI by closing and opening the PR should restart the "PR" job based on PR'ing to current master and this time it should pass.

**Comment by henryiii** [comment]
But travis is p.a.i.n.f.u.l.l.l.l.l.y slow as it [dies](https://www.jeffgeerling.com/blog/2020/travis-cis-new-pricing-plan-threw-wrench-my-open-source-works).

File: plumbum/commands/processes.py [hunk 1]
```diff
@@ -354,11 +354,8 @@ def iter_lines(proc,
 
     assert mode in (BY_POSITION, BY_TYPE)
 
-    encoding = getattr(proc, "custom_encoding", None)
-    if encoding:
-        decode = lambda s: s.decode(encoding).rstrip()
-    else:
-        decode = lambda s: s
+    encoding = getattr(proc, "custom_encoding", None) or 'utf-8'
```
Comment (henryiii) [inline plumbum/commands/processes.py:357]:
```suggestion
    encoding = getattr(proc, "custom_encoding", 'utf-8')
```

Am I missing something or is this the same?

Comment (koreno) [reply plumbum/commands/processes.py:357]: it's not the same in case `proc.custom_encoding` is explicitly `None`

**Comment by henryiii** [comment]
Sorry for the delay, needed lots of CI work, we now have a unified GHA suite and completes in a few minutes and has better coverage. :)

**Comment by henryiii** [comment]
Thanks!

**Pull request merged**